src/moonlink_backend/tests/test_backend.rs

```diff
@@ -959,4 +978,315 @@ mod tests {
             .await;
         assert!(res.is_err());
     }
+
+    /// Reconnect resumes replication (single table)
+    #[cfg(feature = "test-utils")]
```

Comment:
Are you sure we want to guard it with "test-util" attribute? Is it actually tested in CI?